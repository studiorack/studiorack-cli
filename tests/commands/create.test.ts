import { expect, test } from 'vitest';
import { cli } from '../shared';
import path from 'path';
import { RegistryType } from '@open-audio-stack/core';

const APP_DIR: string = 'test';
const PROJECT_DIR: string = path.join(APP_DIR, 'create-project');

test('Create package', async () => {
  expect(cli(RegistryType.Plugins, 'create', PROJECT_DIR)).toMatchSnapshot();
});
