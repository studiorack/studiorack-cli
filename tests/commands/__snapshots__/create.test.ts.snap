// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Create package 1`] = `
"[34m?[39m [1mOrg id[22m [2m(org-name)[22m[21G
[?25h"
`;
